auth/server/src/main/java/com/seagates3/dao/ldap/PolicyLdapStore.java

```diff
@@ -243,4 +244,108 @@ void delete (Object policyObj) throws DataAccessException {
       throw new DataAccessException("Failed to delete the policy.\n" + ex);
     }
   }
+
+ public
+  List<Policy> findByIds(Map<String, Object> dataMap)
+      throws DataAccessException {
+    String accountName = dataMap.get("accountName") != null
+                             ? (String)dataMap.get("accountName")
+                             : "";
+    List<String> policyIds = dataMap.get("policyIds") != null
+                                 ? (List<String>)dataMap.get("policyIds")
+                                 : null;
+
+    String[] attrs = {
+        LDAPUtils.POLICY_ID,                 LDAPUtils.PATH,
+        LDAPUtils.POLICY_CREATE_DATE,        LDAPUtils.POLICY_UPDATE_DATE,
+        LDAPUtils.DEFAULT_VERSION_ID,        LDAPUtils.POLICY_DOC,
+        LDAPUtils.POLICY_NAME,               LDAPUtils.IS_POLICY_ATTACHABLE,
+        LDAPUtils.POLICY_ARN,                LDAPUtils.POLICY_ATTACHMENT_COUNT,
+        LDAPUtils.POLICY_PERMISSION_BOUNDARY};
+    // String ldapBase = LDAPUtils.BASE_DN;
```

Comment:
Remove the commented code if not needed in future

Reply:
Done